app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(17, 4)) == 17
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(30, 6) == [5, 5, 5, 5, 5, 5]
```

Comment:
This is a good positive test case. It checks if the function can split an integer into equal parts when the integer is divisible by the number of parts.